### DISCUSSION TITLE
NVIDIA old driver 11040

### DISCUSSION THREAD

**Discussion by PGTech85** [lllyasviel/Fooocus#2473] [Q&A] [answered]
So I get the
`RuntimeError: The NVIDIA driver on your system is too old (found version 11040). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver.`

So doing a search I think what I need is PyTorch 2.0.1 CUDA 11.7 
I install Anaconda and run
`conda install pytorch==2.0.1 torchvision==0.15.2 torchaudio==2.0.2 pytorch-cuda=11.7 -c pytorch -c nvidia`

So I test it
`>>> import torch
>>> torch.cuda.is_available()
True
>>>`

But I still get the RuntimeError why?

**Comment by PGTech85** [comment]
Ok nevermind I just found out I have CUDA 11.4

**Reply by acastry** [reply]
so what did you do ? How to move forwad with CUDA 11.4 ?

**Reply by mashb1t** [reply, marked as answer]
@acastry please use CUDA 11.8+ as this is what Fooocus is built for. Versions below 11.8 are not officially supported and are most likely not compatible.
